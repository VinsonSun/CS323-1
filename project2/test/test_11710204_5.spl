int test_11710204_5()
{
  float a = 5.6;
  int b = a;
}
