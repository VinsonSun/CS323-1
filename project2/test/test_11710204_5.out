Error type B at Line 1: Missing closing parenthesis ')'
Error type A at Line 3: unknown lexeme `
